docker-app/worker_wrapper/wrapper.py

```diff
@@ -203,13 +204,26 @@ def _run_docker(
 
         client = docker.from_env()
 
+        extra_envvars = {}
+        pgservice_file_contents = ""
+        for secret in self.job.project.secrets.all():
+            if secret.type == Secret.Type.ENVVAR:
+                extra_envvars[secret.name] = secret.value
+            elif secret.type == Secret.Type.PGSERVICE:
+                pgservice_file_contents += f"\r\n{secret.value}"
```

Comment:
Are you sure about the windows line ending ? Does a test case cover multiple secrets ?
```
[conf1]
host=127.0.0.1
...
password=postgres\r
[conf2]
host=127.0.0.1
...
```

Reply:
It doesn't. Good idea.